newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -86,7 +87,17 @@ const ExtensionInstallDialog = ({
   const canInstallExtension = !isInstalling && isCompatible;
   const onInstallExtension = React.useCallback(
     () => {
-      if (canInstallExtension) onInstall();
+      if (canInstallExtension) {
+        if (alreadyInstalled) {
+          const answer = Window.showConfirmDialog(
+            'This extension is already in your project, this will install the latest version. Do you want to continue?'
```

Comment:
This extension is already in your project, this will install the latest version. You may have to do some adaptations to make sure your game continue to work. Do you want to continue?